tools/ok_validator/src/okh.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel
+
+from .validate import OKValidator
+
+
+class OKH(BaseModel, extra="allow"):
+    title: str
+    bom: list[str]
```

Comment:
as per https://okh.makernet.org/form, I think bom field is a string, not an array of strings